lib/floe/workflow/runner/kubernetes.rb

```diff
@@ -78,6 +78,15 @@ def output(runner_context)
           runner_context["output"] = output
         end
 
+        def wait(contexts, timeout: 5)
+          contexts = [contexts] unless contexts.kind_of?(Array)
+          refs     = contexts.map { |ctx| ctx["container_ref"] }
```

Comment:
Does `Array.wrap(contexts) meet our needs

```suggestion
          refs     = Array.wrap(contexts).map { |ctx| ctx["container_ref"] }
```

Reply:
:+1: No activesupport so no `Array.wrap()` but we can use `Array(contexts).map`